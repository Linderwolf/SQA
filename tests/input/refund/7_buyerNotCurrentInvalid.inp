admin
refund
user3
exit