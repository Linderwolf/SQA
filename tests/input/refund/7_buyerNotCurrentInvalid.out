Welcome to Vapour!
Please enter your username to log in: 
Welcome admin! Please enter a transaction: 
Please enter the buyer's username (the recipient of the refund): 
Error: user3 not found. Please enter a valid user.